ui/reactAngularAdaptor.js

```diff
@@ -1,22 +1,25 @@
 import {react2angular} from 'react2angular';
-import AddAppointmentContainer from './react-components/containers/AddAppointmentContainer.jsx';
+import AppointmentContainer from './react-components/containers/AppointmentContainer.jsx';
 
 angular.module('bahmni.appointments')
-.component('reactAddAppointment', react2angular(AddAppointmentContainer));
+.component('reactAddAppointment', react2angular(AppointmentContainer));
 
 // Used this component to pass a callback function to react component
 angular.module('bahmni.appointments').component('reactAddAppointmentWrapper',{
-    template: '<react-add-appointment on-back="onBack" set-view-date="setViewDate">',
+    template: '<react-add-appointment on-back="onBack" set-view-date="setViewDate" appointment-uuid="appointmentUuid" is-recurring="isRecurring">>',
     controller: reactAddAppointmentController
 });
 
-reactAddAppointmentController.$inject = ['$scope', '$state'];
-function reactAddAppointmentController($scope, $state) {
+reactAddAppointmentController.$inject = ['$scope', '$state', '$stateParams'];
+function reactAddAppointmentController($scope, $state, $stateParams) {
     $scope.onBack = function () {
         $state.go('^', $state.params, {reload: true});
     };
 
     $scope.setViewDate = function (date) {
-      $state.params.viewDate = date;
+        $state.params.viewDate = date;
     };
+    console.log($state.current.url);
```

Comment:
can we remove this console log?

Reply:
You will get conflicts with Popup card Alekhya and Siva are working on, please coordinate.

Reply:
Yup. removed